htdocs/postdeploy.php

```diff
@@ -0,0 +1,68 @@
+<?php
```

Comment:
This script is a little scary being in htdocs and doing this.. is it possible that it can at least check that the request is coming from Heroku? Perhaps by checking that the request is coming from localhost, since from what it looks like the success_url is relative.. otherwise, does heroku provide some sort of authentication token that can be verified?